util/kube/kube.go

```diff
@@ -120,3 +123,56 @@ func GetAppInstanceLabel(un *unstructured.Unstructured, key string) string {
 	}
 	return ""
 }
+
+// isNullList checks if the object is a "List" type where items is null instead of an empty list.
```

Comment:
See comment for `GenerateManifest` - we can remove `isNullList` and `FromUnstructured` methods.